graphics/libfive/Portfile

```diff
@@ -0,0 +1,163 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+# libfive does not have releases; the master branch is recommended.
+# See https://libfive.com/download/
+github.setup        libfive libfive 9aa20520497636659b8d5bda25057631837e998a
+version             20210509
+revision            0
+
+categories          graphics math cad
+platforms           darwin
+license             MPL-2
+maintainers         {gmail.com:szhorvat @szhorvat} openmaintainer
+
+homepage            https://libfive.com/
+
+description         Library for 3D solid modeling
+
+long_description    libfive is a software library and set of tools for solid modeling, \
+                    especially suited for parametric and procedural design. It is \
+                    infrastructure for generative design, mass customization, and \
+                    domain-specific CAD tools.
+
+checksums           rmd160  0c28f01b9ccd296fb046c3f0e3c4b3b8b3b6b3e1 \
+                    sha256  7497c1e2cd0c1427cc0d0720d1b19dc9e8d7451156101fef17cfd1b005413f8a \
+                    size    877936
+
+depends_lib-append  port:eigen3 \
```

Comment:
doesn’t eigen3 only contain headers - if so, it should probably only a build dependency... or?